This patch is specific to Oracle branding and will not be submitted upstream.

--- openscap-1.2.1/xsl/xccdf-branding.xsl.~1~	2015-01-10 03:44:55.684466847 -0800
+++ openscap-1.2.1/xsl/xccdf-branding.xsl	2015-01-14 09:35:05.077056778 -0800
@@ -43,24 +43,67 @@
     1) Round all positions to 1 decimal place
     2) Remove inkscape, RDF, DC and other metadata
     -->
-
-<svg xmlns="http://www.w3.org/2000/svg" version="1.1" width="52" height="52" id="svg2"><g transform="matrix(0.75266991,0,0,0.75266991,-17.752968,-104.57468)" id="g32"><path d="m 24.7,173.5 c 0,-9 3.5,-17.5 9.9,-23.9 6.8,-6.8 15.7,-10.4 25,-10 8.6,0.3 16.9,3.9 22.9,9.8 6.4,6.4 9.9,14.9 10,23.8 0.1,9.1 -3.5,17.8 -10,24.3 -13.2,13.2 -34.7,13.1 -48,-0.1 -1.5,-1.5 -1.9,-4.2 0.2,-6.2 l 9,-9 c -2,-3.6 -4.9,-13.1 2.6,-20.7 7.6,-7.6 18.6,-6 24.4,-0.2 3.3,3.3 5.1,7.6 5.1,12.1 0.1,4.6 -1.8,9.1 -5.3,12.5 -4.2,4.2 -10.2,5.8 -16.1,4.4 -1.5,-0.4 -2.4,-1.9 -2.1,-3.4 0.4,-1.5 1.9,-2.4 3.4,-2.1 4.1,1 8,-0.1 10.9,-2.9 2.3,-2.3 3.6,-5.3 3.6,-8.4 0,0 0,-0.1 0,-0.1 0,-3 -1.3,-5.9 -3.5,-8.2 -3.9,-3.9 -11.3,-4.9 -16.5,0.2 -6.3,6.3 -1.6,14.1 -1.6,14.2 1.5,2.4 0.7,5 -0.9,6.3 l -8.4,8.4 c 9.9,8.9 27.2,11.2 39.1,-0.8 5.4,-5.4 8.4,-12.5 8.4,-20 0,-0.1 0,-0.2 0,-0.3 -0.1,-7.5 -3,-14.6 -8.4,-19.9 -5,-5 -11.9,-8 -19.1,-8.2 -7.8,-0.3 -15.2,2.7 -20.9,8.4 -8.7,8.7 -8.7,19 -7.9,24.3 0.3,2.4 1.1,4.9 2.2,7.3 0.6,1.4 0,3.1 -1.4,3.7 -1.4,0.6 -3.1,0 -3.7,-1.4 -1.3,-2.9 -2.2,-5.8 -2.6,-8.7 -0.3,-1.7 -0.4,-3.5 -0.4,-5.2 z" id="path34" style="fill:#12497f"/></g></svg>
+<svg
+     version="1.1"
+     xmlns="http://www.w3.org/2000/svg"
+     xmlns:xlink="http://www.w3.org/1999/xlink"
+     width="100px" height="49px"
+     viewBox="0 0 100 49" preserveAspectRatio="none">
+   <g>
+<image alt="Oracle Solaris" width="100" height="49"
+ xlink:href="data:image/png;base64,
+iVBORw0KGgoAAAANSUhEUgAAAMgAAABhCAMAAACklR6KAAAAn1BMVEX////lAAAAAADn5+ff398M
+DAzxm53tRlPqKzPpIi771dnnAA/nABLnAAv2tLf5wMT72dvtYGZ3d3frQUk3NzfxfYW+vr6IiIif
+n5/++fqurq7y8vI/Pz/Pz89gYGD2qLCSkpL85OZubm4jIyP+8fNTU1MbGxtKSkouLi6AgID5x8ru
+b3TvWGTtU1vqO0HwZ3HzlJrxhovpGSP0rKzveXu2qy5AAAAG10lEQVR4nO1Z2ZaiSBQkwZVFUUtk
+ERHFjSoVLf//2yZuAsrmdHlq5tgPGQ+t5MaNvEtcqyVJQEBAQEBAQEBAQEBAQEBA4E0wvztlBGY2
+45hJp4pBWNnuqJdbUB2Uorh6aBQO8G9tYQY1KC9PQikaBMMXaITnHquif1ZTGwe1KcaOldPjEwYv
+UZlcMDEq274l88hYT200Ih5dqyY4UtI9j5qXN6GDTUb5pfRoJE5KRNdrkzqfu1/EDWtY77s4Fk9o
+YWnjLZTMkcEmjZad60aEkjocfXaCn/K4ME3Htmu/ADzrzJg6GZHSHC5Og9WPA5wOMzScMTIfY8MT
+0zUcUtjWBYFnRJwvZpARhdcYHQRncovPP42tgGka0ztxrBYQJz0w0RNORNPPYXFycASTz0ekq108
+gwt7uEmlJexzUNwWOk+JRDcdPE5ByQY6TL0kHae2vBFqH6+cRLXxcAQmpyEnok3Lc2YXkT64W3Fh
+BrsluND+oDjUC+omPCEy6MGIS1MRiJ5VhiocBBb7aiId8XtvJCINYGeSnxAg1iYxMk27BxcMKzD9
+I5HwiPMuP7z5JzD7zOjFjVMhsgQGwugakRhG50TMETOuSUTRxD8BJyGidS8/I0JF79i0/AWoTDPO
+zVMUH7jWBo9EMFTv3L8jzdXcMzwzUcVY8+00E7lcDXYe1vCSi4Yw89I8haAy+p0mj8T93GT+3TB4
+CQs/4YcpXWwMtSiWsD8ROcOIq66VgHB4yUeI7Pxu60QMdk2yHDGDOzpHqr8ZN5MyKQtvRIjB6LAh
+NOSzMUubiYxghFEBYz8XQgIV16R5ijSde4SIhNOCUqFeH9PXRB2IZfad6wml/cseGaVKVsb0teRH
+2hq35qkIdvWCPEecAPKWITWXb0eaPrQRwaVRcJmQ6dMLOXLTDGhOJUNeTH4TSnZqdmKISIGQ5MlO
+DtKzbmWavQVZbRij73uzd4YqUtlFMTAaVOQZkUEPyf676kulCS5p1BGaKekIrl83rpfwsfobHjjF
+6ukeD2BKkTboP3FJMxGHdCT5JZOY5bWmwiNh/HaLggip0IuCrVJ2XxB1mqHrj7i7ROQp1m1g8kTZ
+O6i/15/2Ik/gTHHx7KvyEyMMRtSLfkplZUffojEtp00+40mN3ChUAtYfUs+Ia7jUTOZE6vUsonMh
+Jb+jcqJ+j117RVCHi3IUVoiguyN+Z26KQwGUNSJOjpwc1BGFSOuXDj2p5kgzjNIgjU8i80Tvqxrx
+ateCGlMrf6iw7BZJqSwWBBERB0upI+e9iXapBiXve9GeEEut8suq48AjdHIFA4fardry/ssOCrq1
+syHdAye9d/pJ9FjrBPS6/jcFFrtLSPEwDPcg++a0+ovvaJJH6+AXFgWn6nhzC/iviNTOuVvEKImz
+SHaGvW63KJnO4KvbnZxj9TjpfjV0uOFl0p1Qx+iEg9uxcOYx5rPdKr6yN4Xx5aswfHqi0wICAgIC
+AgLP4Sl/M7yfE1nJ7b8W8tz/OZHW34xXPCIgICAgICDwEhT3sBrbrfxhvFgtLVJez7Ld1mOZb9uu
+IqXjBNeip5a9pHbD8+1Vtu1dcKy5TNi79GR98Ad5DAbKQd5b93XKGMN8jbJK18gfliO5a9n2pNY4
+26a8iQXQ2sjbg2/L8g436+/l7cx3Qe2gSMpMnj+I+Du5LS/pykFpvRiv1nyL+9EGkYMsbyxrIxOp
+N8Gz5e0Sn5YsL6XWIjUdpn64ZSKevV5/bHnXitkFhRyst1Ii/kZewRf4WLzNJTkRxXJ9yZ9nwWFt
+5YPnFYm0VvJ4uZMzmhsitHwQ2XEGju/67/OIu5U/ZmmWenZ7a/NRMFopJSIWSMBhiLjcI7j/TRZa
+ykJuL9w35geBJzGCHiZ7S3mdEUGQtIpEMDX3kf2USNixne92a+6XNEfsNs5oz2dv/ZnhuLAJ2LVg
+bVq76kQQWUuqaWvXIyJtTn6p5ERQtva8bLm//G+2X8J3x1sUKsm+e6QaWtZaPli+PefVbCzvZy5t
+QRHLiFCSzXZtedN69o7/G557IEMcpMrYc9ftNDr8dTnZ4at2qhQIJxCh6oWMQapwItZsSQT8BZeW
+94AqLpUgKMhBau3SegQK+1L5RahlaNtORsQ70AcRcZapgEApdy/8VeG/BcqvvGopuExEFT1sLMWH
+Qqy4IO5XM8D1XFx1RuigZESkmbzNyi+ag52lKMv1O6U9v2weVHBFFkHUouQPK+uQSiDNf1ge5PKh
+I3tyxn3b2xwCKEskenuVmuBY1EjtbLpYz94tOFz/sHDTymotNpa33ByIlrXauY6/WkCEPJeuY26/
+WUoEBAQEBAQEBAQEBAQEBAT+H/wDTtmvpnnKESQAAAAASUVORK5CYII="/> </g> </svg>
 </xsl:template>
 
 <xsl:template name="xccdf-report-header">
     <nav class="navbar navbar-default" role="navigation">
-        <div class="navbar-header" style="float: none">
+	 <div class="navbar-header" style="width:100%; height:100%; padding-bottom:5px; float:none">
             <a class="navbar-brand" href="#">
                 <xsl:call-template name="xccdf-branding-logo"/>
             </a>
-            <div><h1>OpenSCAP Evaluation Report</h1></div>
+	<div style="padding-left:30px; text-align:left;"><h2>Compliance Report</h2></div>
         </div>
     </nav>
 </xsl:template>
 
 <xsl:template name="xccdf-report-footer">
     <footer id="footer">
-        <div class="container">
+        <!--
+	 <div class="container">
             <p class="muted credit">
                 Generated using <a href="http://open-scap.org">OpenSCAP</a>
                 <xsl:if test="$oscap-version">
@@ -68,22 +110,24 @@
                 </xsl:if>
             </p>
         </div>
+	 -->
     </footer>
 </xsl:template>
 
 <xsl:template name="xccdf-guide-header">
     <nav class="navbar navbar-default" role="navigation">
-        <div class="navbar-header" style="float: none">
+	 <div class="navbar-header" style="width:100%; height:100%; padding-bottom:5px;float:none">
             <a class="navbar-brand" href="#">
                 <xsl:call-template name="xccdf-branding-logo"/>
             </a>
-            <div><h1>OpenSCAP Security Guide</h1></div>
+	<div style="padding-left:30px; text-align:left;"><h2>Security Guide</h2></div>
         </div>
     </nav>
 </xsl:template>
 
 <xsl:template name="xccdf-guide-footer">
     <footer id="footer">
+	<!--
         <div class="container">
             <p class="muted credit">
                 Generated using <a href="http://open-scap.org">OpenSCAP</a>
@@ -92,6 +136,7 @@
                 </xsl:if>
             </p>
         </div>
+	 -->
     </footer>
 </xsl:template>
 
